src/app/components/listing/confirm/confirm-listings.jsx

```diff
@@ -283,15 +291,17 @@ function ChplConfirmListings(props) {
                     { listing.displayStatus }
                     { beta
                       && (
-                        <>
-                          <br />
+                        <div>
                           <Button
                             onClick={() => { setErrors(listing.errors); setWarnings(listing.warnings); }}
                             disabled={!(listing.errors?.length !== 0 || listing.warnings?.length !== 0)}
+                            variant='text'
+                            color='primary'
+                            className={classes.messageButton}
                           >
-                            See messages
+                            See messages <FeedbackIcon color='primary' fontSize='small' className={classes.iconSpacing}/>
```

Comment:
Our style linter will say there should not a JSX element on the same line as anything else, so would want this to look something like:
```suggestion
                            See messages
                            {' '}
                            <FeedbackIcon color='primary' fontSize='small' className={classes.iconSpacing}/>
```

Also, the `'`s will need to be `"`s